docs/userGuide/reusingContents.md

```diff
@@ -226,11 +211,26 @@ When setting the `id` of a fragment, be careful not to clash with heading anchor
 
 </box>
 
+**Attributes:**
+* **`src`**: specify the source file path.
+* **`inline`** (optional): make the included result an inline element. (wrapped in `<span>` tag). e.g.,
+  ```html
+  <include src="../docs/summary.md" inline />
+   ```
+* **`optional`** (optional): include the file/fragment only if it exists i.e., there will be no error message if the file/fragment does not exist. e.g.,
+  ```html
+  <include src="UserStories.md" optional />
+  ```
+* **`trim`** (optional): remove leading and trailing whitespace and newlines from the document before including.
+  ```html
+  <include src="frament.md" trim />
```

Comment:
`frament.md` -> `UserStories.md#epic`? (since this attribute is most likely to be used by fragments, I have taken the opportunity to add in a fragment example).